pkgs/applications/networking/browsers/telescope/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, autoreconfHook
+, bison
+, libevent
+, libressl
+, ncurses
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "telescope";
+  version = "0.3";
```

Comment:
Is there any reason we are not using `0.3.1`? Seems that just got released the other day, would be great if we could upgrade to that.

Reply:
Uh, i'm having troubles with fetchFromGithub and cannot fetch the newest version.
With fetchurl works fine :D